cli/src/athenapdf.js

```diff
@@ -22,6 +22,11 @@ if (!process.defaultApp) {
     process.argv.unshift("--");
 }
 
+function collect(val, memo){
```

Comment:
Rather than constructing an array, we should probably be constructing a dictionary / map as that is the data structure that `requestHeaders` requires:

```
const collectDict = (keyVal, dict) => {
    const [key, ...values] = keyVal.split("=");
    dict[key] = values.join("=");
    return dict
}
```

That way, we'll save ourselves from the complex logic that happens later on.